internal/scanner/scanner.go

```diff
@@ -58,8 +66,14 @@ func (s *Scanner) ScanNamespace(namespace string) error {
 	// TODO this is for debugging, it should be remove in future steps!
 	for _, resource := range auditableResources {
 		fmt.Println("Policies: ")
-		for _, policy := range resource.Policies {
+		for i, policy := range resource.Policies {
 			fmt.Println(policy.GetName())
+			url, err := s.resourcesFetcher.GetPolicyServerURLRunningPolicy(context.Background(), &resource.Policies[i])
```

Comment:
In the previous version, due the way that the argument of the `GetPolicyServerURLRunningPolicy` method should be passed, that was needed to avoid a security check issue from the linter. However, I've changes the argument to a proper type and now I can pass `policy` directly.